microsoft-edge/webview2/concepts/process-model.md

```diff
@@ -16,35 +16,46 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, I
+
+For example, the number of render processes varies based on the following conditions.  
+
+<!-- TODO:  which previous content?  -->  
+*   Use of the website isolation feature (described in the previous content) in the WebView2 Runtime.  
+*   The number of distinct disconnected origins rendered in instances of WebView2 using the same User Data Folder.  
+
+
+## WebView2 Runtime processes and the User Data Folder
+All processes in a WebView2 Runtime processes collection are tied to the browser process, which in turn is associated with a single User Data Folder (UDF).  If an application makes use of multiple User Data Folders, a WebView2 Runtime processes collection will be created for each of these User Data Folders. A User Data Folder can be shared by multiple applications, but applications choosing to do this should carefully consider the implications on performance and management. See [Manage the user data folder][WebView2ManageUDF] for more information.
 
 :::image type="complex" source="../media/process-model-2.png" alt-text="Process 2" lightbox="../media/process-model-2.png":::
    Process 2  
 :::image-end:::    
 
-A browser process is associated with several renderer processes.  A WebView 2 instance creates a browser process to service frames.  A browser process may be associated with multiple frames.  A browser process may be associated with different instances of WebView2.  The number of render processes varies based on the following conditions.  
+To make use of multiple User Data Folders, a WebView2 application needs to create different `CoreWebView2Environment` objects, each configured with a different User Data Folder value. When the first `WebView2` instance is created over a given User Data Folder (through the configured `CoreWebView2Environment`), the browser process for the WebView2 Runtime processes collection associacted to that UDF will be started, and all additional processes will be managed by its lifetime.
 
-*   Use of the website isolation feature in your browser.  
-*   The number of distinct disconnected origins rendered in associated instances of WebView2.  
-    
-The website isolation browser feature is described in the previous content. 
-<!--todo:  which previous content?  -->  
+<!-- TODO: update with profile info -->
+The `CoreWebView2Environment` represents a User Data Folder and the collection of processes associated to it.  A given render process is not associated to single `CoreWebView2` instance, as it can serve frames in multiple `CoreWebView2` instances using the same User Data Folder, depending on website isolation as previously described.  
```

Comment:
```suggestion
The `CoreWebView2Environment` represents a User Data Folder and the collection of processes associated to it.  A given renderer process is not associated to single `CoreWebView2` instance, as it can serve frames in multiple `CoreWebView2` instances using the same User Data Folder, depending on website isolation as previously described.  
```